putio.py

```diff
@@ -9,6 +9,15 @@
 import requests
 import iso8601
 
+KB = 1024
+MB = 1024 * KB
+
+# Downloads are broken up into ranges of this size
+RANGE_SIZE = 10 * MB
+
+# Downloads are written and flushed after a chunk of this size
+CHUNK_SIZE = 8 * KB
```

Comment:
I haven't done any performance tests for this yet, no. I'm not sure how useful they would be given that everyone's machines are different and something like this would be very dependent on the performance of the receiving machine. In my case, for example, I have a RaspberyPi managing downloads, so it's not fast and it's quite memory constrained.

http://stackoverflow.com/questions/23369625/ideal-chunk-size-for-python-requests

I'll do some performance tests to get more info on this. Unit tests for this sort of thing are near-impossible without a mess of mock objects and functions.